src/modules/core/components/TransactionList/TransactionDetails.tsx

```diff
@@ -68,7 +76,10 @@ const UserDetails = ({
 );
 
 const ColonyDetails = ({
-  colony: { displayName: colonyDisplayName, colonyAddress },
+  colony: {
+    displayName: colonyDisplayName,
+    colonyAddress,
+  } = ColonyRecord().toJS(),
```

Comment:
If I don't call `.toJS()` on the record, TS complains about not being able to destructure it, but the code itself works even without this call.